docs/units-and-conventions.rst

```diff
@@ -118,3 +118,23 @@ If we had a fully sampled grid of :math:`{\cal V}_{u,v}` values, then the operat
     I_{l,m} = U V (\Delta u)(\Delta v) \mathtt{iFFT}({\cal V}_{u,v})
 
 For more information on this procedure as implmented in MPoL, see the :class:`~mpol.gridding.Gridder` class and the source code of its :func:`~mpol.gridding.Gridder.get_dirty_image` method. When the grid of :math:`{\cal V}_{u,v}` values is not fully sampled (as in any real-world interferometric observation), there are many subtleties beyond this simple equation that warrant consideration when synthesizing an image via inverse Fourier transform. For more information, consult the seminal `Ph.D. thesis <http://www.aoc.nrao.edu/dissertations/dbriggs/>`_ of Daniel Briggs.
+
+
+"Packed" Image Cubes
+--------------------
+
+Internally, data is represented by Cubes, or arrays of three dimensions. In the case of MPoL, the dimensions of an MPoL cube represent Declination (DEC, axis one), Right Ascension (RA, axis two), and Velocity/Frequency (axis three). Initially, the Normal Cube, or Sky Cube, is the orientation of an image on the sky. Due to this, the RA increases to the *left*. It is desirable to flip the RA axis to obtain a traditional scaling with RA increasing to the *right*.
+
+We now have a flipped version of the Sky Cube (referred to as the Flip Cube). The next step is to "pack" the cube. In this application, this is carried out by applying ``torch.fft.fftshift()`` to the Flip Cube. This method acts only along the DEC and RA axes (axis one and axis two) leaving axis three untouched and creating a Packed Cube, or Image Cube. This method is represented internally as ``mpol.utils.sky_cube_to_packed_cube()``. 
```

Comment:
The Sky Cube itself is kinda tricky. In general, astronomical images are usually plotted with north up and east to the left. "l" = RA and "m" = DEC. So as it appears on the sky, we have the "l" axis increasing to the left. This is a little strange compared to the way we normal look at x,y axis, where x increases to the right.